docs/configure/processes.md

```diff
@@ -1,78 +1,55 @@
-
-{{% /tab %}}
-{{< /tabs >}}
+{{< alert title="In this page" color="note" >}}
+{{% toc %}}
+{{< /alert >}}
 
 ## Configure a process
 
 Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
-Click the **+** icon next to your machine {{< glossary_tooltip term_id="part" text="part" >}} in the left-hand menu and select **Process**.
-Your process is automatically created with a name (`id` in JSON) like **process-1** and a card matching that name on the **CONFIGURE** tab.
-Navigate to that card.
+Click the **+** icon next to your machine part in the left-hand menu and select **Process**.
 
-Then fill in the following fields:
+In the process configuration panel, configure the attributes for your process:
 
 <!-- prettier-ignore -->
-| Attribute (Builder Mode) | Attribute (JSON) | Type    | Required? | Description |
-| ------------------------ | ---------------- | ------- | --------- | ----------- |
-| Executable               | `name`               | string  | **Required** | The command you want to execute when your machine connects to the server. On many operating systems, you can find the executable path of commands by running with `which <command-name>`. |
-| Arguments                | `args`               | string  | Optional     | Arguments to follow the command. |
-| Working directory        | `cwd`                | string  | Optional     | Where you want the process to execute. Defaults to the directory where `viam-server` executes. |
-| Logging                  | `log`                | boolean | Optional     | Toggle logging of errors and other messages on or off. Default: `false`. |
-| Execute once             | `one_shot`           | boolean | Optional     | Toggle whether to execute the command just once or keep running the process indefinitely.<ul><li>If true, the process executes once at `viam-server` startup. Until the process completes, `viam-server` startup is blocked and the robot appears offline in the [Viam app](https://app.viam.com), so this should only be used for quick processes.</li><li>If false, the process continues to run. If the process crashes, it automatically restarts. It does not block `viam-server`. Default: `false`.</li></ul> |
-
-Click **Save** in the upper right corner of the screen to save your config.
+| Attribute | Type | Required? | Description |
+| --------- | ---- | --------- | ----------- |
+| Executable (`name`) | string  | **Required** | The command you want to execute when your machine connects to the server. On many operating systems, you can find the executable path of commands by running `which <command-name>`. |
+| Arguments (`args`) | string  | Optional | Arguments to follow the command. |
+| Working directory (`cwd`) | string  | Optional | Where you want the process to execute. Defaults to the directory where `viam-server` executes. |
+| `username` (not available in builder mode) | string | Optional | Example: `"username": "ubuntu"`. |
```

Comment:
@zaporter-work I was not able to confirm that the `username` attribute does anything. My assumption is that it would allow us to run the process as a user?

On my pi I added a process with this config:
```
{
      "env": {
        "SVC_API_KEY": "VALUE"
      },
      "username": "pi",
      "id": "process-2",
      "name": "python",
      "args": [
        "username.py"
      ],
      "cwd": "/home/pi",
      "log": true,
      "one_shot": false
    }
```
and this little python program:

```
import os
import getpass
import time

while True:
  print(1)
  print(time.sleep(1))
  raise ValueError(getpass.getuser())
```

I can't see anytihng printed at all and what gets raised always indicates that the process runs as `root`. 

Does `username` do somethign else?

Reply:
Thank you so much for testing this! 

I have just confirmed that with the newest RDK, this now works.

A slightly easier way to test that this works is via the `whoami` command:
```json
  "processes": [
    {
      "username": "pi",
      "id": "process-1",
      "name": "whoami",
      "log": true
    }
  ],
```